azure-pipelines/build.yml

```diff
@@ -0,0 +1,122 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+# Please make sure to check all the converted content, it is your team's responsibility to make sure that the pipeline is still valid and functions as expected.
+# This pipeline will be extended to the MicroBuild template
+trigger: none
+name: $(Date:yyyyMMdd).$(Rev:r)
+variables:
+- name: ApiScanClientId
+  value: 8cce479d-aee7-48ad-a21a-547253d6f599
+- name: ApiScanEnabled
+  value: true
+- name: ApiScanSecret
+  value: ''
+- name: ApiScanTenant
+  value: 72f988bf-86f1-41af-91ab-2d7cd011db47
+- name: ArtifactServices.Symbol.AccountName
+  value: microsoftpublicsymbols
```

Comment:
AFAIK, this doesn't work anymore. You may want to start with the WPF Behaviors first as the classic pipeline for that one has been upgraded more recently